src/components/tab-nav/tab-nav.tsx

```diff
@@ -56,6 +56,13 @@ export class TabNav {
    */
   @Prop({ reflect: true }) syncId: string;
 
+  /**
+   * Specifies the component's selected tab-title.
+   *
+   * @readonly
+   */
+  @Prop({ mutable: true }) selectedTabTitle: HTMLCalciteTabTitleElement = null;
```

Comment:
For consistency, should it be `selectedTitle`? I don't think we we add the component name in our props. 

For example `selectedItems` rather than `selectedTreeItems`, and `filteredItems` instead of `filteredComboboxItems`

Reply:
Changed!